astropy/convolution/utils.py

```diff
@@ -286,3 +286,48 @@ def discretize_integrate_2D(model, x_range, y_range):
             values[j, i] = dblquad(model, x[i], x[i + 1],
                                    lambda x: y[j], lambda x: y[j + 1])[0]
     return values
+
+def profile_memory_usage_convolve_fft():
+    """
+    Utility to profile the memory usage of convolve_fft
+
+    Requires memory_profiler and timeit
+    """
```

Comment:
Would be nice though to have more information on what exactly this is profiling.  I think it should also take some optional arguments, such as the list of input sizes to profile (with a default similar to the current one).  Not all the imports need to be in-line either.  At most just `timeit` and `memory_profiler`.